### PR TITLE
completed task 11 for feedback

### PR DESCRIPTION
**Pull request by BeckBob**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -402,4 +402,44 @@ describe("GET /api/users", () => {
...
+            expect(body.msg).toBe("Bad Request")
+        })
+    })
+    test("404: Not Found when topic doesn't exist i.e. /api/articles?topic=sharon", () =>{
+        return request(app)
+        .get("/api/articles?topic=sharon")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("Not Found")
+        })
```
Comment (MRombi) [inline __tests__/endpoints.test.js:443]: We should add a test for when the topic exists but there are no articles related, for our test data the topic is paper 🙂

File: models/topics.model.js [hunk 2]
```diff
@@ -3,4 +3,15 @@ const db = require("../db/connection");
 exports.getTopicsArray = () => {
     return db.query("SELECT * FROM topics")
     .then(({ rows }) => { return rows });
-}
\ No newline at end of file
+}
+
+exports.checkTopicExists = (query) => {
+const {topic} = query
+console.log(topic)
```
Comment (MRombi) [inline models/topics.model.js]: We should remove console.log 🙂

**Review by MRombi — CHANGES_REQUESTED** [review]
Great job with this ticket! You are very close, just one extra test! Please, action on the feedback I left and resubmit 🙂

**Review by MKD01 — APPROVED** [review]
Really good work, I am happy for you to merge this in.

**Pull request merged**